src/views/ReviewConfirm.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div>
+    <header>
+      <h2>5. Review</h2>
+    </header>
+
+    <v-card flat class="review-confirm-container">
+      <section>
+        Review/confirm component goes here.
+      </section>
+    </v-card>
+  </div>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Vue } from 'vue-property-decorator'
+
+@Component({})
```

Comment:
Same as above, just curious if you would prefer this!

Reply:
I don't think I have a preference.